source/stylesheets/base/_typography.scss

```diff
@@ -39,26 +39,43 @@ a {
   }
 }
 
-.link-light {
-  color: $blue;
-  background: url('../../images/link-background-light.svg') no-repeat center left;
-  background-size: contain;
-  padding: 3px 20px;
-}
-
-.link-medium {
-  color: $blue;
-  background: url('../../images/link-background-medium.svg') no-repeat center center;
+@mixin gen-link($color, $imgURL, $position, $imgURLHover) {
+  color: $color;
+  background: url($imgURL) no-repeat $position;
   background-size: contain;
   padding: 3px 20px;
   &:hover {
     a {
       color: $white;
     }
-    background: url('../../images/link-background-medium-hover.svg') no-repeat center center;
+    background: url($imgURLHover) no-repeat center center;
   }
 }
 
+.link-light {
+  @include gen-link(
+    $blue,
+    '../../images/link-background-light.svg',
```

Comment:
Do you need the `../../images` bit? I know in Rails the asset pipeline figures these things out for you sometimes. You might be able to pass `link-background-light.svg.`

Reply:
It seems to need them when I tried taking it out.